internal/pkg/utils/testproject/project.go

```diff
@@ -104,7 +119,10 @@ func GetTestProject(t *testing.T, envs *env.Map) *Project {
 	p.setEnv(`TEST_KBC_STORAGE_API_HOST`, p.Project.StorageAPIHost())
 	p.setEnv(`TEST_KBC_STORAGE_API_TOKEN`, p.Project.StorageAPIToken())
 	p.logf(`■ ️Initialization done.`)
-	return p
+	return p, func() {
+		unlockFn()
+		cancelFn()
```

Comment:
Chápu to dobře že se tady mají zavolat obě, jo? 🙂

Reply:
jj, teda idealne v opacnom poradi ... najprv zastavime vsetky operacie `cancelFn` a tak odomkneme projekt.

Reply:
ok